docs/reference/docs/reference/delete_fragment.rst

```diff
@@ -0,0 +1,8 @@
+Article's Fragment
+==================
+
+This feature moves the removed fragment inputs in another DOM and thereafter
+this allows the cancellations of removed fragments.
+After the deletion of a fragment, each deleted fragment item in the fragment list
+contains a reset button (to undo the deletion) and the fragment title. This action triggers undo of
+the deletion of the fragment so the deleted fragment inputs (and elements) are moved to their original place.
```

Comment:
Please comply with the sf docs standards:

> [Each line should break approximately after the first word that crosses the 72nd character (so most lines end up being 72-78 characters);](http://symfony.com/doc/current/contributing/documentation/standards.html)

Also, the docs job fails in Travis, you can fix it by adding a reference to this page in `docs/index.rst`